examples/text_classification/download.py

```diff
@@ -0,0 +1,23 @@
+import logging
```

Comment:
Since download/extract_archive functions are more orthogonal now, we could add this file under examples directory. They are not so relevant to text_classification things.

Reply:
The only reason it's specific to text classification is because it pulls the URL from the text_classification package. But we can make the URL an argument.

Reply:
I think use an URL argument makes more sense. It's just a download API and we want people to use it separately.